ui.apps/src/main/content/jcr_root/apps/groovyconsole/clientlibs/js/console.js

```diff
@@ -173,7 +180,7 @@ var GroovyConsole = function () {
 
           $('#schedule-job-text').text('Scheduling...');
 
-          $.post(CQ.shared.HTTP.getContextPath() + '/bin/groovyconsole/jobs.json', {
+          $.post(GroovyConsole.getContextPath() + '/bin/groovyconsole/jobs.json', {
```

Comment:
create var for this and interpolate in js string?

Reply:
it's used all over the places, not sure if that will make it easier than this :)

Reply:
if not more readable we should use this indeed